app/controllers/admin_controller.rb

```diff
@@ -6,7 +6,19 @@ def index
     @msa_components = MsaComponent.all
     @services = Service.all
     @certificates = Certificate.all
-    @teams = Team.all
+    teams = Team.all
+    @relying_parties = []
+    @identity_providers = []
+    @other = []
+    teams.each do |team|
+      if team.team_type == "rp"
+        @relying_parties << team
```

Comment:
Would have extract this into a method, but I think this is much better

`    @teams = Team.all
    @relying_parties, @identity_providers, @other = Team
      .all.group_by(&:team_type).values_at(TEAMS::RP, TEAMS::IDP, TEAMS::OTHER)`